<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_EditPlayers</class>
 <widget class="QDialog" name="Dlg_EditPlayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Player database</string>
  </property>
  <widget class="QPushButton" name="btnAddPlayer">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Add new player</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="MoveListWidget" name="listItems">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Use selected player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSortName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sort by Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSortTeam">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sort by Team</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoPlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="favTeams">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image: url(:/Versus/expand.png);
border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="teamName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logoTeam">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MoveListWidget</class>
   <extends>QListWidget</extends>
   <header>MoveListWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnCancel</tabstop>
  <tabstop>logoPlayer</tabstop>
  <tabstop>logoTeam</tabstop>
  <tabstop>playerName</tabstop>
  <tabstop>teamName</tabstop>
  <tabstop>favTeams</tabstop>
  <tabstop>btnAddPlayer</tabstop>
  <tabstop>btnSortName</tabstop>
  <tabstop>btnSortTeam</tabstop>
  <tabstop>listItems</tabstop>
  <tabstop>btnOK</tabstop>
 </tabstops>
 <resources>
  <include location="../RC/Versus.qrc"/>
 </resources>
 <connections/>
</ui>
